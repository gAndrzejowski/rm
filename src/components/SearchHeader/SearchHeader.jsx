import React from 'react';
import styles from '../Heading/Heading.scss';
import SearchBox from '../SearchBox/SearchBox';
import PageTitle from '../PageTitle/PageTitle';

const SearchHeader = () => (
    <header>
        <div className={styles.top}>
            <PageTitle/>
        </div>
        <SearchBox/>
    </header>
);
export default SearchHeader;
