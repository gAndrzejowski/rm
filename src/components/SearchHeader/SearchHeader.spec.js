import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import SearchHeader from './SearchHeader';

describe('Search Header', () => {
  it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<SearchHeader />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
