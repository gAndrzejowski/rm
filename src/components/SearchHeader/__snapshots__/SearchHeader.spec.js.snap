// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Header renders correctly 1`] = `
<header>
  <div
    className="top"
  >
    <PageTitle />
  </div>
  <Connect(SearchBox) />
</header>
`;
